operator/pkg/manifest/client.go

```diff
@@ -151,7 +152,7 @@ func (client *Client) GetPods(namespace string, params map[string]string) (*v1.P
 		req.Param(k, v)
 	}
 
-	res := req.Do()
+	res := req.Do(context.TODO())
```

Comment:
initialization should use `context.Background()`  `TODO` is if we don't know or cannot send ctx yet.

Here and elsewhere. I see that TODO is used everywhere.

Reply:
IMO, TODO is more suitable, Background is more often used to initiate a complete context.